tests/views/test_update_event_date.py

```diff
@@ -142,8 +142,6 @@ def test_put_update_event_date_update_bad_request(self, mock_request):
                                     data=self.update_event_form, follow_redirects=True)
 
         self.assertEqual(response.status_code, 200)
-        self.assertIn("Must be after MPS Thursday 11 Oct 2018 23:00".encode(), response.data)
```

Comment:
Can we check for anything else in this test or does the flash stop us doing a unit test on it?

Reply:
not as far as I'm aware as the message displayed isn't stored in a variable? Could be wrong

Reply:
I'm sure you can test for the flash, I think you'll need to put some data inside the put mock request

Reply:
Couldn't you just mock Flash and validate it was called with specific params ?

Reply:
I don't think it's even being called, looks like its getting a `ValueError` which I don't think it should be getting but i'm not too sure. Not entirely sure what the test should be doing.

Reply:
Chris has added tests and changed some to check for the flash message occurring. The tests before were just testing for a 200 which is right but you wouldn't know which path it was going down